docs/assets/design-tokens.css

```
@@ -82,6 +82,28 @@
   --da-color-icon-chart:      #db2777;
   --da-color-icon-chart-bg:   #fce7f3;
 
+  /* ── Dependency-graph node category colors (#444 dep-graph migration) ──
+     Distinct semantic axis from icon-*/journey-*/mode-*: these classify
+     dependency-graph NODE TYPES, not icons/phases/modes. 3-role set per
+     category — bg = saturated fill (selected node) + node stroke;
+     light = soft fill (unselected node); text = node label color.
```

Stylelint (17.12.0)

[error] Unknown word light (CssSyntaxError)

(CssSyntaxError)